packages/react-components/src/components/gif.tsx

```diff
@@ -244,6 +244,7 @@ const Gif = ({
     return (
         <Container
             href={noLink ? undefined : gif.url}
+            data-gph-gif={gif.id}
```

Comment:
@pshoniuk this is handy for tests, I've always used `[href="${gifId}"]` as the selector, which works but we have the no links option and then there's no a tag and no href 

Just wondering if it we can spell out `giphy` here instead of `gph`. I know it's more characters but might as well keep it consistent with the classNames such as `giphy-grid`

Reply:
Awesome, already updated

Reply:
I used the GPH prefix out of habit;) Recently I worked with RN and mobile, which uses this prefix. But your suggestion is excellent because it keeps the code base consistent.